css/style.css

```diff
@@ -1,3 +1,77 @@
-h1 {
-  color: salmon;
+body {
+  font-family: sans-serif;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #5c5f60;
+  background-color: #edeeec;
+}
+
+#container, section, header, .spot-div, .spot {
```

Comment:
Perhaps instead of using a lot of different selectors here, you could define a "flex" class that can be added to all of these elements in your HTML...